### DISCUSSION TITLE
Request: After UBL 3-point tilt, either reset or return M117 status message

### DISCUSSION THREAD

**Discussion by nullstalgia** [Jyers/Marlin#887] [Ideas]
Howdy

Coming at you again with a simple thing for that polish, and I didn't feel like it deserved an Issue.

Reason being, by default, Cura doesn't send anything there, so anyone who is missing a command after the tilt will always see "
Tilting 3/3"

You could also do a "Tilting complete," but that feels a little half-baked. :)

**Comment by gothcha** [comment]
I added an M117 message ‘Printing’ to the cura start code after the G29 J to sort this ‘issue’
